packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.tsx

```diff
@@ -17,10 +17,12 @@ import DescriptionIcon from '@material-ui/icons/Description'
 import { useMulti } from '../../lib/crud/withMulti';
 import MessageIcon from '@material-ui/icons/Message'
 import Button from '@material-ui/core/Button';
+import EditIcon from '@material-ui/icons/Edit';
 import * as _ from 'underscore';
 import { DatabasePublicSetting } from '../../lib/publicSettings';
+import { Select, MenuItem } from '@material-ui/core';
 
-export const defaultModeratorComments = new DatabasePublicSetting<Array<string>>('defaultModeratorComments', ["yMHoNoYZdk5cKa3wQ"])
+export const defaultModeratorComments = new DatabasePublicSetting<Array<object>>('defaultModeratorComments', [{label:"Not Good Enough", id:"yMHoNoYZdk5cKa3wQ"}])
```

Comment:
@jimrandomh or @Discordius, am I supposed to specify the structure of the objects here? I'm not actually sure what to google to learn about the syntax here for the <Array<object>> thingy

Later in the file, there are type-errors for the "label" and "id" parts of the object. 

![image](https://user-images.githubusercontent.com/3246710/106101659-0ad54300-60f3-11eb-878e-aabb525b64bb.png)

I clicked "quick fix" on the label error, and then clicked on the "Declare property 'label'" option, but I don't see what actually changed or whether it was important. (Fake edit: looks like it added a property to the "object" class in some obscure file which a) looks incorrect and b) looks like it was gitigored anyway)

Not sure if I'm supposed to care about this or not.

Reply:
Yep, I don't think you should ever use the `object` type. Specifically you should first create a type of the shape: 

```
type moderatorCommentRecord = {label: string, id: string}
```

And then do 

```
new DatabasePublicSetting<Array<moderatorCommentRecord>>
```

I prefer the alternative Array syntax: 

```
new DatabasePublicSetting<moderatorCommentRecord[]>
```

Reply:
Minor nitpick: the type name should be capitalized, ie,
```
type ModeratorCommentRecord = {label: string, id: string}
```
(We haven't been fully consistent about this, though.)